calico.go

```diff
@@ -58,6 +59,8 @@ func cmdAdd(args *skel.CmdArgs) error {
 		EnableDebugLogging()
 	}
 
+	glog.Infof("Conf data: %s\n", conf)
```

Comment:
I think we want to add logging to the CNI plugin.  It's been a bit difficult to figure out what is going on in some of my testing without debug logs.

If we're using glog, when we should be using the `.V()` method to set verbosity levels of the logs.

e.g
```go
glog.V(3).Infof(...)
```